spec/abci++/abci++_app_requirements.md

```diff
@@ -486,7 +497,7 @@ They enforce certain limits in the blockchain, like the maximum size
 of blocks, amount of gas used in a block, and the maximum acceptable age of
 evidence. They can be set in
 [`InitChain`](./abci%2B%2B_methods.md#initchain), and updated in
-[`FinalizeBlock`](./abci%2B%2B_methods.md#finalizeblock).
+[`EndBlock`](./abci%2B%2B_methods.md#endblock).
 These parameters are deterministically set and/or updated by the Application, so
 all full nodes have the same value at a given height.
 
```

Comment:
Could you please remove the `TimeoutParams.*` below?
We are not backporting this from v0.36.x.

Also the `SynchronyParams.*`,  which are PBTS-related, which I think will be backported soon, but won't make it for v0.37.x.

Actually, I just checked `proto/tendermint/types/params.proto` and we should include `ValidatorParams.*`, and `VersionParams.*`

Reply:
Done!.